src/NzbDrone.sln

```diff
@@ -90,192 +90,361 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "LogentriesNLog", "Logentrie
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CurlSharp", "ExternalModules\CurlSharp\CurlSharp\CurlSharp.csproj", "{74420A79-CC16-442C-8B1E-7C1B913844F0}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SocksWebProxy", "ExternalModules\SocksWebProxy\SocksWebProxy\SocksWebProxy.csproj", "{EF64B71A-6512-460E-8F0A-BCD456FF9461}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Org.Mentalis", "ExternalModules\SocksWebProxy\Org.Mentalis\Org.Mentalis.csproj", "{4C3F4F57-A121-44BA-97CE-B5F30ED6A735}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
 		Debug|x86 = Debug|x86
+		Mono|Any CPU = Mono|Any CPU
 		Mono|x86 = Mono|x86
+		Release|Any CPU = Release|Any CPU
 		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{FAFB5948-A222-4CF6-AD14-026BE7564802}.Debug|Any CPU.ActiveCfg = Debug|x86
```

Comment:
We aren't planning to support Any CPU since it makes shipping native binaries for x64, more complicated/error prone.